docs/getting-started/how-to/use-released-outputs-in-github-codespaces/index.md

```diff
@@ -0,0 +1,30 @@
+Whilst it's possible to execute a project pipeline entirely inside a secure environment,
+in practice it's often more convenient to execute the final stage outside.
+This is because the final stage frequently involves carefully crafting figures and tables for publication,
+making many small adjustments that would otherwise entail multiple round-trips to the OpenSAFELY Jobs site.
+
+Executing the final stage of a project pipeline outside a secure environment is only possible when the outputs from the previous stage have been released to the OpenSAFELY Jobs site.
+[Released outputs](../../../releasing-files-intro.md) have been subject to statistical disclosure control and have been reviewed by two trained OpenSAFELY output checkers.
+
+To copy released outputs to a Codespace,
+first download them from the OpenSAFELY Jobs site to your computer,
+then drag-and-drop them from your computer to the Codespace.
+Similarly, to copy figures and tables from a Codespace,
+drag-and-drop them from the Codespace to your computer.
+
+Copying released outputs to a Codespace,
+and copying figures and tables from a Codespace,
+are examples of **sharing results**.
+The "[Datasets Used](https://www.opensafely.org/policies-for-researchers/#datasets-used)" section
+of the "Policies for Researchers" page on the OpenSAFELY website contains more information about sharing results.
+
+!!! warning
+    Never commit released outputs, figures, or tables to your project's repository.
+    Doing so would contravene the "Policies for Researchers" page on the OpenSAFELY website.
+
```

Comment:
This feels like it would be very easy to do by accident. Should we recommend/ship a released outputs folder with appropriate gitignore and tell people to drag and drop there?

Reply:
That sounds like a good approach 👍